src/e2e/java/teammates/e2e/cases/AdminSearchPageE2ETest.java

```diff
@@ -37,8 +37,8 @@ public void allTests() {
         InstructorAttributes instructor = testData.instructors.get("instructor1OfCourse1");
         AccountAttributes instructorAccount = testData.accounts.get("instructor1OfCourse1");
 
-        ______TS("Typical case: Search student google id");
-        String searchContent = student.getGoogleId();
+        ______TS("Typical case: Search student email");
+        String searchContent = student.getEmail();
```

Comment:
Google ID was never a searchable field; email is. This passed before because the google ID happened to be the same as the email minus domain.